web-registry/src/pages/CreditBatchIssuance.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import Box from '@mui/material/Box';
+
+import {
+  BatchBasicsForm,
+  BatchBasicsFormValues,
+} from 'web-components/lib/components/form/BatchBasicsForm';
+import OnBoardingSection from 'web-components/lib/components/section/OnBoardingSection';
+import Stepper from 'web-components/lib/components/stepper';
+
+const steps = ['Credit Basics', 'Recipients', 'Review', 'Finished'];
+
+const CreditBatchIssuance: React.FC = () => {
+  const [activeStep, setActiveStep] = useState(0);
+
+  const handleBasicsSubmit = (values: BatchBasicsFormValues): Promise<void> => {
+    // console.log(JSON.stringify(values)); TODO: set values to parent context
+    setActiveStep(1);
+    return Promise.resolve();
+  };
+
+  return (
+    <>
+      <Stepper
+        sx={{ mw: 240 }}
+        steps={steps}
+        activeStep={activeStep}
+        onStepClick={(stepIndex: number) => setActiveStep(stepIndex)}
+      />
+      <OnBoardingSection title={steps[activeStep]} formContainer>
+        <Box minHeight="50vh">
+          {activeStep === 0 && <BatchBasicsForm submit={handleBasicsSubmit} />}
```

Comment:
@victordelval Here is a simple implementation of a single-page flow. I think it would be fairly simple to hold the "parent" form at this level, and allow the sub-forms to populate it on submit. Hopefully this is similar to what you were planning.

Reply:
This is a perfect starting point, thanks!